x-pack/plugins/reporting/server/export_types/png/types.d.ts

```diff
@@ -14,9 +14,7 @@ export interface JobParamsPNG extends CreateJobBaseParams {
 }
 
 // Job payload: structure of stored job data provided by create_job
-export interface ScheduledTaskParamsPNG extends ScheduledTaskParams<JobParamsPNG> {
-  basePath?: string;
-  browserTimezone: string;
```

Comment:
`basePath` should not be a job param (it is referenced at the time of job execution)

`browserTimezone` is provided by the interface being extended